huxley/api/permissions.py

```diff
@@ -1,10 +1,14 @@
-# Copyright (c) 2011-2015 Berkeley Model United Nations. All rights reserved.
+# Copyright (c) 2011-2016 Berkeley Model United Nations. All rights reserved.
 # Use of this source code is governed by a BSD License (see LICENSE).
 
-from rest_framework import permissions
+import json
+
+from django.http import QueryDict
 
 from huxley.core.models import Assignment, Delegate
 
+from rest_framework import permissions
```

Comment:
tiny nit: this goes directly under the `django.http` import, since they're both libraries